docs/cmdline-opts/cookie-jar.d

```diff
@@ -19,9 +19,12 @@ written. The file will be written using the Netscape cookie file format. If
 you set the file name to a single dash, "-", the cookies will be written to
 stdout.
 
+This option specifies a file that cookies are save to. Not read from. To tell
```

Comment:
s/save/saved/ but I like the way --cookies says it better, and in reverse:

The file specified with [-c, --cookie-jar](https://curl.se/docs/manpage.html#-c) is only used for output. No cookies will be read from the file. To read cookies, use the [-b, --cookie](https://curl.se/docs/manpage.html#-b) option. Both options can specify the same file.

Reply:
agreed!